python/pyspark/sql/functions/builtin.py

```diff
@@ -5385,6 +5385,24 @@ def countDistinct(col: "ColumnOrName", *cols: "ColumnOrName") -> Column:
 
     .. versionchanged:: 3.4.0
         Supports Spark Connect.
+    
+    Examples
```

Comment:
Hmm `countDistinct` is an alias for `count_distinct`. I don't think we need an example for one.